packages/circuit-ui/components/ComparisonTable/components/PlanTable/PlanTable.tsx

```diff
@@ -119,10 +119,21 @@ export const PlanTable = forwardRef<HTMLTableElement, PlanTableProps>(
       // opt for progressive enhancement
       // eslint-disable-next-line compat/compat
       const headerSizeObserver = new ResizeObserver((entries) => {
+        /* account for sticky plan picker on mobile */
         const planPickerHeight = isPlanPickerVisible
           ? (isMobile ? 80 : 0) + (isTablet ? 16 : 0)
           : 0;
-        setSectionOffset(entries[0].contentRect.height + planPickerHeight);
+        /* account for sticky top navigation, if it exists */
+        const topNavigationHeight = Number(
+          getComputedStyle(document.documentElement)
```

Comment:
Getting the value from the document root ignores the cascade, i.e. any values that are set on a parent of the ComparisonTable other than the `html` element. 

```suggestion
          getComputedStyle(tableRef.current)
```